docker-compose.yml

```diff
@@ -93,7 +93,7 @@ services:
     command: minio --compat server data
 
   solr:
-    build: "https://github.com/vulekamali/solr-dokku.git"
+    build: "https://github.com/TomaszKolek/solr-dokku.git"
```

Comment:
This needs to be deleted before merged, also PR that is created in main solr-dokku repo needs to be merged.

Reply:
https://github.com/vulekamali/solr-dokku/pull/7

Reply:
merged
```suggestion
    build: "https://github.com/vulekamali/solr-dokku.git"
```